hyperspy/models/model2d.py

```diff
@@ -120,7 +120,7 @@ def __init__(self, signal2D, dictionary=None):
         self.axes_manager.events.indices_changed.connect(
             self._on_navigating, [])
         self._channel_switches = np.ones(
-            self.axes_manager.signal_shape, dtype=bool
+            self.axes_manager.signal_shape[::-1], dtype=bool
```

Comment:
Better to use the following:?
```suggestion
            self.axes_manager._signal_shape_in_array, dtype=bool
```
which is equivalent?
https://github.com/hyperspy/hyperspy/blob/6163e539bc0e63470993550bcb84c3801d9bfa88/hyperspy/axes.py#L1582-L1584